torch/onnx/symbolic_opset17.py

```diff
@@ -15,5 +15,19 @@
     SequenceMap
 """
 
+from torch.onnx import symbolic_helper
+
 # EDITING THIS FILE? READ THIS FIRST!
 # see Note [Edit Symbolic Files] in symbolic_helper.py
+
+
+@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
+def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
```

Comment:
```suggestion
def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
```

cudnn_enable is a boolean. Since it is not used, we can annotate as "none" in parse args

Reply:
Should we use None or keep it as original (use v) in this case?

Reply:
If "none" is a common usage, I will add it into doc of parse args.

Reply:
Done